core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -266,8 +286,11 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
       return;
     }
 
-    [this.activeSuggestionId, this.activeContribution] = (
-      this.remainingContributions.pop());
+    let lastContributionsDetail = (
```

Comment:
Perhaps it would be better to change this.remainingContributions to contain actual dicts with the relevant keys, not 2-element lists (since the latter doesn't come with the proper context).

Reply:
Done